src/components/MonitorLog/AdditionalSpeciesTab.jsx

```diff
@@ -153,135 +158,139 @@ const AdditionalSpeciesTab = ({ showHeader, isDisabled, species }) => {
-              isDisabled={isDisabled}
-              onChange={val => setValue(`${FORM_PREFIX}injuredCount`, parseInt(val, 10))}
-              defaultValue={getValues(`${FORM_PREFIX}injuredCount`) || 0}
-            >
-              <NumberInputField />
-              <NumberInputStepper>
-                <NumberIncrementStepper />
-                <NumberDecrementStepper />
-              </NumberInputStepper>
-            </NumberInput>
-          </FormControl>
-
-          <Text fontWeight="600" fontSize="xl">
-            Beach Cast
+            <Link to="/species">
+              <Text color="#2B6CB0" fontWeight="500" maxW="100vw">
+                Open Species Catalog
+              </Text>
+            </Link>
+            <Link to="/species">
+              <Text color="#2B6CB0" fontWeight="500" maxW="100vw">
+                <FiExternalLink />
+              </Text>
+            </Link>
+          </HStack>
+        </>
+      )}
+      <VStack align="left" spacing="29px">
+        {showHeader && (
+          <Text fontWeight="600" fontSize="2xl">
+            Non-Listed Species
           </Text>
-          <FormControl>
-            <NumberInput
-              mb="50px"
-              min={0}
-              isDisabled={isDisabled}
-              onChange={val => setValue(`${FORM_PREFIX}beachCast`, parseInt(val, 10))}
-              defaultValue={getValues(`${FORM_PREFIX}beachCast`) || 0}
-            >
-              <NumberInputField />
-              <NumberInputStepper>
-                <NumberIncrementStepper />
-                <NumberDecrementStepper />
-              </NumberInputStepper>
-            </NumberInput>
-          </FormControl>
-        </VStack>
-      </Stack>
-    </VStack>
+        )}
+        <FormControl>
+          <Grid marginTop="20px" minH="200px" templateColumns="repeat(6, 1fr)" gap="150">
+            <GridItem colSpan={3}>
+              <Box overflow="hidden" border="1px solid darkgray" rounded="md" mb="4">
+                <Accordion as={Table} allowToggle reduceMotion>
+                  <Thead w="100%" bg="#4E4E4E" borderColor="gray.200">
+                    <Tr>
+                      <Th w="8%" bgColor="none" />
+                      <Th
+                        w="65%"
+                        fontWeight={600}
+                        color="#FFFFFF"
+                        textTransform="capitalize"
+                        fontSize=".8em"
+                      >
+                        Species
+                      </Th>
+                      <Th
+                        bgColor="none"
+                        fontWeight={600}
+                        color="#FFFFFF"
+                        textTransform="capitalize"
+                        fontSize=".8em"
+                      >
+                        Total
+                      </Th>
+                      <Th />
+                    </Tr>
+                  </Thead>
+                  {species.length === 0 && (
```

Comment:
🚫 **[eslint]** <[react/prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'species.length' is missing in props validation